### PR TITLE
Feature/eth removal

### PR DESCRIPTION
**Pull request by ice-hades**

### CODE REVIEW

File: scripts/syncLocalizations.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        fs.writeFileSync(
+          jsonFilePath,
+          JSON.stringify(existingJson, null, 2) + '\n',
+        );
+        checkAndAddNewline(jsonFilePath);
+      }
+    }
+  });
+
+  console.log('JSON files updated successfully.');
```
Comment (ice-ares) [inline scripts/syncLocalizations.js:70]: remove

File: scripts/syncLocalizations.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+// Example usage: node syncLocalizations.js <path_to_new.json> <path_to_json_files_directory>
+const args = process.argv.slice(2);
+
+if (args.length !== 2) {
+  console.log(
```
Comment (ice-ares) [inline scripts/syncLocalizations.js:77]: remove

File: scripts/syncLocalizations.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+function updateJsonFiles(newJsonPath, jsonFilesDirectory) {
+  if (!fs.existsSync(newJsonPath)) {
+    console.error(`Error: new.json does not exist at ${newJsonPath}`);
+    return;
+  }
+
+  if (!fs.statSync(jsonFilesDirectory).isDirectory()) {
+    console.error(`Error: Directory does not exist at ${jsonFilesDirectory}`);
```
Comment (ice-ares) [inline scripts/syncLocalizations.js:42]: remove

File: scripts/syncLocalizations.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+function checkAndAddNewline(filePath) {
+  const fileContents = fs.readFileSync(filePath, 'utf8');
+  if (!fileContents.endsWith('\n')) {
+    fs.appendFileSync(filePath, '\n');
+  }
+}
+
+function updateJsonFiles(newJsonPath, jsonFilesDirectory) {
+  if (!fs.existsSync(newJsonPath)) {
+    console.error(`Error: new.json does not exist at ${newJsonPath}`);
```
Comment (ice-ares) [inline scripts/syncLocalizations.js:37]: remove

File: scripts/syncLocalizations.js (hunk 2 above)
Comment (ice-hades) [reply scripts/syncLocalizations.js:77]: wait. that is script file. not an app

File: scripts/syncLocalizations.js (hunk 3 above)
Comment (ice-hades) [reply scripts/syncLocalizations.js:42]: wait. that is script file. not an app

File: scripts/syncLocalizations.js (hunk 1 above)
Comment (ice-hades) [reply scripts/syncLocalizations.js:70]: wait. that is script file. not an app

File: scripts/syncLocalizations.js (hunk 4 above)
Comment (ice-hades) [reply scripts/syncLocalizations.js:37]: wait. that is script file. not an app

**Review by ice-ares — APPROVED** [review]

**Review by ice-hector — APPROVED** [review]

**Pull request merged**